### PR TITLE
Scrappy's first commits!

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: src/site/models/orgcommitteedetails.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php
+
+// No direct access.
+defined( '_JEXEC' ) or die;
+
+jimport( 'joomla.application.component.modeladmin' );
+
+class SwaModelOrgCommitteeDetails extends SwaModelForm {
+    /**
```
Comment (addshore) [inline src/site/models/orgcommitteedetails.php]: odd indenting here, but no need to worry too much!

File: src/site/views/orgcommitteedetails/view.html.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            throw new Exception( implode( "\n", $errors ) );
+        }
+        
+        // If not logged in
+	if( $this->user->id === 0 ) {
+            $url = 'index.php?option=com_users';
+            $url.= '&return=' . base64_encode( JURI::getInstance()->toString() );
+            $app->redirect( JRoute::_( $url, false ) );
+	}
+
```
Comment (addshore) [inline src/site/views/orgcommitteedetails/view.html.php]: Need to make sure the user is a committie member before showing the view.

You can see an example of getting a member object here https://github.com/SWAuk/com_swa/blob/master/src/site/views/universityeventattendees/view.html.php#L39
And checking if they are on a club committie here (although we would want to check org_committee)
https://github.com/SWAuk/com_swa/blob/master/src/site/views/universityeventattendees/view.html.php#L43

This is using the getMember method which is in the base Model class used.
Can be seen at https://github.com/SWAuk/com_swa/blob/orgcommitteedetails/src/site/ModelForm.php#L13 ! :)
The model for orgcommitteedetails is already using this base class so it should all work!

**Comment by addshore** [comment]
I'm going to go ahead and merge this as it is, still needs a submit button though!

**Comment by oshotton** [comment]
Working on a submit button, got most of the code there just need to tweak it to work.

**Comment by addshore** [comment]
Wahaha, the submit button is there already there! :O
The only todo is the method in the controller that the submit button calls!

**Pull request merged**